### PR TITLE
Start an experiment to measure the impact of ads renderOnIdle feature.

### PR DESCRIPTION
**Pull request by lannka**
For #31436

### CODE REVIEW

**Review by calebcordry — APPROVED** [review]

**Comment by lannka** [comment]
Please take another look. I switched to mendel.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -366,9 +378,27 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // holdback branch sends non-numeric value.
       return false;
     }
-    // eslint-disable-next-line no-undef
-    const defaultRange = IDLE_VIEWPORT_RANGE_3 ? 3 : 12;
-    return vpRange || defaultRange;
+
+    if (vpRange) {
+      return vpRange;
```
Comment (calebcordry) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:383]: Is it ok if we are using `vpRange` or `this.performance_ = null` but we are still reporting exp id / control id in ads metrics?

**Review by powerivq — APPROVED** [review]

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -447,6 +479,16 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         isTrafficEligible: () => true,
         branches: Object.values(PTT_EXP_BRANCHES),
       },
+      {
+        experimentId: IDLE_CWV_EXP,
+        isTrafficEligible: () => () => {
```
Comment (calebcordry) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:484]: is this intended to be a nested function?

Comment (lannka) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: nope :-). updated

**Review by calebcordry — APPROVED** [review]

**Pull request merged**